pkg/kubelet/network/exec/exec.go

```diff
@@ -33,7 +33,7 @@ limitations under the License.
 //                formatted output indicating the pod's IPAddress(v4/v6). An empty string value or an erroneous output
 //                will mean the container runtime (docker) will be asked for the PodIP
 //                e.g. {
-//                         "apiVersion" : "v1beta1",
+//                         "apiVersion" : "v1",
```

Comment:
@WIZARD-CXY Because I know PodNetworkStatus struct represents version "v1" which is defined at https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/network/plugins.go#L60, so I changed the doc. But I'm not familiar with the module of networkplugin, and maybe I didn't notice the code of line 172, so I don't change the code.